### PR TITLE
MAGEDOC-3969 Added note to MTF to use MFTF

### LINKED ISSUE #5033: MTF pages should notify users that framework is outdated
# Topic clarification/correction request

## Topic Link

https://devdocs.magento.com/guides/v2.3/mtf/mtf_introduction.html

## Description
This link dedicated to explain how our outdated Functional Testing Framework works and this misleading community, because we should advertise MFTF current framework.

## Possible solutions
Add header to all MTF doc pages which say that this framework is outdated.
In text it would be nice to have link to MFTF documentation and link to project dedicated to convert all outdated tests to new framework: https://github.com/magento/magento-functional-tests-migration

### PR DESCRIPTION
**Pull request by dobooth**
## Purpose of this pull request

This pull request (PR) adds a note to MTF pages to inform them to use MFTF.

https://jira.corp.magento.com/browse/MAGEDOC-3969

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.3/mtf/mtf_introduction.html

Closes #5033

### CODE REVIEW

File: guides/v2.2/mtf/mtf_installation.md [hunk 1]
```diff
@@ -3,6 +3,10 @@ group: functional-testing-framework-guide
 title: Installation of the Functional Testing Framework Entities
 ---
 
+{: .bs-callout-info}
+The Magento Testing Framework (MTF) has been superseded by the Magento Functional Testing Framework ([MFTF][]).
```
Comment (shrielenee) [inline guides/v2.2/mtf/mtf_installation.md]:
```suggestion
The Magento Testing Framework (MTF) is superseded by the Magento Functional Testing Framework ([MFTF][]).
```

File: guides/v2.2/mtf/mtf_installation.md [hunk 2]
```diff
@@ -3,6 +3,10 @@ group: functional-testing-framework-guide
...
+While the MTF is still functional, all MTF tests are being ported over to the MFTF. We recommend using the MFTF for testing.
```
Comment (shrielenee) [inline guides/v2.2/mtf/mtf_installation.md]:
```suggestion
While the MTF is still functional, all MTF tests are ported over to the MFTF. We recommend using the MFTF for testing.
```

Comment (shrielenee) [reply guides/v2.2/mtf/mtf_installation.md]: Keep those "be" verbs outta here!

Comment (shrielenee) [reply guides/v2.2/mtf/mtf_installation.md]: Please make these changes on the other version pages in this PR.

File: guides/v2.3/mtf/mtf_installation.md [hunk 3]
```diff
@@ -73,4 +77,7 @@ Find the `mtf` directory.
 Open `<magento2_root_dir>/dev/tests/functional/vendor/magento/mtf/CHANGELOG.md`. The latest version in `CHANGELOG.md` is version of the FTF you installed.
 
 ## Next Steps {#mtf_install_next}
```
Comment (shrielenee) [inline guides/v2.3/mtf/mtf_installation.md]: Also, check your case per section. This one uses camel case, but a previous section uses sentence case. Be consistent, for all versions in this PR.

File: guides/v2.2/mtf/mtf_installation.md (hunk 2 above)
Comment (dobooth) [reply guides/v2.2/mtf/mtf_installation.md]: Will do.

File: guides/v2.3/mtf/mtf_installation.md (hunk 3 above)
Comment (dobooth) [reply guides/v2.3/mtf/mtf_installation.md]: Go it.

File: guides/v2.2/mtf/mtf_installation.md (hunk 2 above)
Comment (dobooth) [reply guides/v2.2/mtf/mtf_installation.md]: Actually, the are still in the process of porting. It's a work in progress.

**Comment by shrielenee** [comment]
I don't need to be an assignee if I just review a PR (only a reviewer). Removed myself.

**Comment by jeff-matthews** [comment]
@shrielenee, our PR and issue project automation will assign you to an issue/PR whenever you review a PR. You can remove yourself, but it'll keep happening.

**Comment by dobooth** [comment]
running tests

**Comment by dobooth** [comment]
running tests

**Pull request merged**